app/presenters/coronavirus_landing_page_presenter.rb

```diff
@@ -1,5 +1,5 @@
 class CoronavirusLandingPagePresenter
-  COMPONENTS = %w(live_stream live_stream_enabled stay_at_home guidance announcements_label announcements nhs_banner sections topic_section country_section notifications find_help).freeze
+  COMPONENTS = %w(live_stream live_stream_enabled stay_at_home guidance announcements_label announcements see_all_announcements_link nhs_banner sections topic_section country_section notifications find_help).freeze
```

Comment:
This isn't part of your change, but I find the use of the term 'components' here and elsewhere confusing, since they're not components in the sense that we think of components. We should have a conversation about that at some point.

Reply:
Yh - we probably need to go through all of the hub code and standardise things.